src/migrations/1697479763911_make-all-words-discoverable.ts

```diff
@@ -0,0 +1,56 @@
+import { Type } from "decentraland-gatsby/dist/entities/Database/types"
+import { ColumnDefinitions, MigrationBuilder } from "node-pg-migrate"
+
+import PlaceModel from "../entities/Place/model"
+
+export const shorthands: ColumnDefinitions = {
+  hidden: {
+    type: Type.Boolean,
+    default: false,
+    notNull: true,
+  },
+}
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.dropIndex(PlaceModel.tableName, ["hidden", "world_name"])
+  pgm.dropIndex(PlaceModel.tableName, ["hidden", "base_position"])
+  pgm.dropIndex(PlaceModel.tableName, ["updated_at"])
+  pgm.dropIndex(PlaceModel.tableName, ["like_rate"])
```

Comment:
Is it okay that these two don't have the `"hidden"` column? They were removed in your suggested commit @braianj

Reply:
Yes if you look when the index has been created, those don't have hidden column either, the hidden is inside the where 

```
pgm.createIndex(PlaceModel.tableName, ["updated_at"], {
    where: "disabled is false and hidden is false and world is false",
  })
  pgm.createIndex(PlaceModel.tableName, ["like_rate"], {
    where: "disabled is false and hidden is false and world is false",
  })

```